packages/storage-documents/src/on-demand-page-scan-result.ts

```diff
@@ -71,3 +71,7 @@ export interface OnDemandPageScanRunResult {
     pageTitle?: string;
     pageResponseCode?: number;
 }
+
+export interface PartialOnDemandPageScanResult extends Partial<OnDemandPageScanResult> {
```

Comment:
likely this interface does not require any members at all since it should provide all `OnDemandPageScanResult` members per `Partial` target functionality

Reply:
I think we do need to require id, since we need it to get the partitionKey and update the document

Reply:
As per other comment I'll drop this type. `Partial<>` as func param type should let use `OnDemandPageScanResult` arbitrary members